plugins/dbnd-airflow-operator/test_dbnd_airflow_operator/test_dag_build.py

```diff
@@ -1,54 +1,113 @@
 import copy
+from typing import Tuple
 
 from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+from dbnd import task, pipeline
+import pytest
+from dbnd._core.errors import MissingParameterError
 
 from test_dbnd_airflow_operator.airflow_home.dags.dag_test_examples import (
     default_args_test,
-    t_A,
-    t_B,
-    t_pipeline,
-    t_two_outputs,
 )
 
 
 class TestFunctionalDagBuild(object):
-    def test_simple_build(self):
-        with DAG(
-            dag_id="test_simple_build", default_args=default_args_test
-        ) as dag_operators:
-            a = t_A("check", 2)
-        assert len(dag_operators.tasks) == 1
-        assert a
-        assert isinstance(a, str)
-
-    def test_simple_wiring(self):
-        with DAG(
-            dag_id="test_simple_build", default_args=default_args_test
-        ) as dag_operators:
-            a = t_A("check")
-            b = t_B(a)
+    def test_task_is_bind_to_dag(self):
+        @task
+        def sample_callable():
+            return True
```

Comment:
I'm in favor of keeping as much in a test as we can. In this way, I don't have to check definitions from other files :) WDYT?